decidim-core/app/models/decidim/action_log.rb

```diff
@@ -49,6 +88,34 @@ def self.for_admin
       where(visibility: %w(admin-only all))
     end
 
+    # All the resource types that are eligible to be included as an activity.
+    def self.public_resource_types
```

Comment:
Just to have it clear, and I know that this is obviously out of the scope of the current developments, as this comes from `ActivitySearch.resource_types`, but this smells like a missing contract/API between core and the spaces/components modules, right?

Reply:
I'll resolve this conversation as I assume it's going to be another PR.